### DISCUSSION TITLE
DC blueretro need help

### DISCUSSION THREAD

**Discussion by hpx1984** [darthcloud/BlueRetro#280] [General]
Dear Sir

Dreamcast P1 BlueRetro can working so i try for connect P2
my esp32 board can't found IO3 PIN ,by checked the BlueRetro diy sch then saw the J3's pin5 is IO3,but this pin on my esp32 board is RX pin
i go ahead for connected P2 PIN1 to IO3 follow BlueRetro diy sch(ESP32 J3-Pin5) and then P1 can't pairing , P2 also can't pairing ,seems esp32 board was damage, but the board can download FW
any way i had one more ESP32 board on hand , need your help for comment which pin on esp32 board i can connect the P2 "IO3"  
![image](https://user-images.githubusercontent.com/76941051/169522787-b4288b3c-fb8a-49fc-bb5e-b5d612e64092.png)
![image](https://user-images.githubusercontent.com/76941051/169523014-7d0a1658-ac44-40a7-8c2e-78e8132aa18f.png)

**Comment by pceslayer** [comment]
RX is I03
